### ISSUE TITLE
For training code

### ISSUE THREAD

**Issue by plzhang4321** [artest08/LateTemporalModeling3DCNN#11]
I have seen this code:
input_vectors=x
norm = input_vectors.norm(p=2, dim = -1, keepdim=True)
input_vectors = input_vectors.div(norm)
Is that code's function same as Batch Normalization?
